runtime/doc/options.txt

```diff
@@ -4583,6 +4583,14 @@ A jump table for the options with a short description can be found at |Q_op|.
 		RedrawDebugRecompose guibg=Red    redraw generated by the
 						  compositor itself, due to a
 						  grid being moved or deleted.
+	    nothrottle	Turn off throttling of the message grid. This is an
+			optimization that joins many small scrolls to one
+			larger scroll when drawing the message area (with
+			'display' msgsep flag active).
+	    invalid	Enable stricter checking (abort) of inconsistencies
+			of the internal screen state. This is mosly
+			useful when running nvim inside a debugger (and
+			the test suite).
```

Comment:
How about renaming 'redrawdebug' to 'debugredraw' (I assume you don't want to cram it into the existing 'debug' option, but at least all debug options could have the same "debug" prefix).

Reply:
'debug' seems to be specifically about showing messages from "expr" callbacks. We have both options beginning with 'debug' (though only 'debug' itself) and with 'redraw'. Maybe 'debugredraw' rolls marginally better off the tongue but is it worth changing it at this point? (I, for one, already have `set rdb` in muscle memory though I guess I'm the only one there)